src/main/java/echo/Server.java

```diff
@@ -0,0 +1,91 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+
+
+public class Server {
+    public static void main(String[] args) throws IOException {
+        runEchoServer();
+    }
+
+    public static ServerSocket serverSocket = null;
+        public static Socket clientSocket = null;
```

Comment:
I'm not good at Java, so excuse my simple question, but is this `null` assignment here needed? I understand that these are declared on top to avoid errors, but would it be enough to do: 

```
    public static ServerSocket serverSocket;
    public static Socket clientSocket;
```

Reply:
I don't think it was needed in that file but now that I've moved those variables to the `main.java` file they can't be passed down if they're not assigned a value. it gives me errors when I try. 
❓